deployments/templates/deployment.yml

```
@@ -38,6 +38,14 @@ spec:
               value: "${ENABLE_ANALYTICS}"
             - name: ANALYTICS_ID
               value: "${ANALYTICS_ID}"
+            - name: AZURE_AUTH_ENABLED
+              value: "${AZURE_AUTH_ENABLED}"
+            - name: AZURE_CLIENT_ID
+              value: "$AZURE_CLIENT_ID"
+            - name: AZURE_REDIRECT_URI
+              value: "$AZURE_REDIRECT_URI"
+            - name: AZURE_AUTHORITY
+              value: "$AZURE_AUTHORITY"      
```

```suggestion
            - name: AZURE_CLIENT_ID
              value: "${AZURE_CLIENT_ID}"
            - name: AZURE_REDIRECT_URI
              value: "${AZURE_REDIRECT_URI}"
            - name: AZURE_AUTHORITY
              value: "${AZURE_AUTHORITY}"
```
minor env var formatting consistency

---

removed